merkledag/errservice.go

```diff
@@ -14,28 +14,34 @@ type ErrorService struct {
 
 var _ ipld.DAGService = (*ErrorService)(nil)
 
+// Add returns an error.
```

Comment:
I find this description confusing, rather than say "an error" something like "returns ErrorService.Err" or maybe just "returns _the_ error" rather than "_an_ error".